pypit/arparse.py

```diff
@@ -477,9 +482,12 @@ def arc_calibrate_lamps(self, v):
         v : str
           value of the keyword argument given by the name of this function
         """
-        allowed = ['ArI', 'CdI', 'HgI', 'HeI', 'KrI', 'NeI', 'XeI', 'ZnI', 'ThAr']
-        v = key_list_allowed(v, allowed)
-        self.update(v)
+        if v == 'None':
```

Comment:
This is fine as is, but I'm tempted to put this into a new function, called something like:
`key_none_list_allowed(v, allowed)`